src/config/routes.js

```diff
@@ -4,6 +4,7 @@ import LoginPage from '../components/login/login.container';
 import TrailMap from '../components/trail-map/trail-map.container';
 import ReportProblem from '../components/report-problem/report-problem.container';
 import ReportProblemIssueType from '../components/report-problem-issue-type/report-problem-issue-type.container';
+import ReportProblemComment from '../components/report-problem-comment/report-problem-comment.container';
```

Comment:
In other words, the index.js file should act as the public interface to the rest of the application.